integrations/spark-scala.properties

```diff
@@ -19,9 +19,10 @@
 # The first Scala version for each Spark version is the one that will
 # appear as a project in IntelliJ.
 
-sparkVersions=3.1,3.2,3.3,3.4
+sparkVersions=3.2,3.3,3.4,3.5
 
 sparkVersion-3.1-scalaVersions=2.12
```

Comment:
we can look up "3.1" in the code. Still documentation and some infra files like this has reference to it.